### PR TITLE
Print dashed lines

### PR DESCRIPTION
**Pull request by rmichaelis**

### CODE REVIEW

File: src/services/print.js [hunk 1]
```diff
@@ -619,6 +619,10 @@ ngeo.Print.prototype.encodeVectorStyleStroke_ = function(symbolizer, strokeStyle
     symbolizer.strokeColor = goog.color.rgbArrayToHex(strokeColorRgba);
     symbolizer.strokeOpacity = strokeColorRgba[3];
   }
+  var strokeDashstyle = strokeStyle.getLineDash();
+  if (goog.isDefAndNotNull(strokeDashstyle)) {
```
Comment (gberaudo) [inline src/services/print.js]: You should use `strokeDashstyle !== undefined` instead of goog.

Comment (rmichaelis) [inline src/services/print.js]: Done. Thanks for the tip.
Why did you decide these last weeks to avoid the use of goog.isDef ?

Comment (gberaudo) [inline src/services/print.js]: We want the code to use as much as possible standard javascript.

**Comment by fredj** [comment]
+1

**Comment by fredj** [comment]
And this correct to have `strokeDashstyle !== null` because `strokeStyle.getLineDash()`return an arraw or null

**Pull request merged**